backend/local/backend_local.go

```diff
@@ -136,14 +136,27 @@ func (b *Local) contextDirect(op *backend.Operation, opts terraform.ContextOpts)
 	}
 	opts.Config = config
 
-	variables, varDiags := backend.ParseVariableValues(op.Variables, config.Module.Variables)
+	var rawVariables map[string]backend.UnparsedVariableValue
+	if op.AllowUnsetVariables {
+		// Rather than prompting for input, we'll just stub out the required
+		// but unset variables with unknown values to represent that they are
+		// placeholders for values the user would need to provide for other
+		// operations.
+		rawVariables = b.stubUnsetRequiredVariables(op.Variables, config.Module.Variables)
+	} else {
+		// If interactive input is enabled, we might gather some more variable
+		// values through interactive prompts.
+		// TODO: Need to route the operation context through into here, so that
```

Comment:
Can you make a comment (here, not necessarily in the code) about why this TODO isn't being done now?

Reply:
I don't have any _grand_ reason for it... just wanted to keep the function signature changes to a minimum here in order to keep the diff minimal, and the context wasn't routed in before either so this hasn't made things any worse than they used to be.